public/assets/styles/footer-style.ts

```diff
@@ -1,6 +1,7 @@
 import { makeStyles } from '@mui/styles';
+import theme from 'src/theme';
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles({
```

Comment:
why didn't you use theme here?
```suggestion
const useStyles = makeStyles(theme)=> ({
```